builders/integrations/indexers/moralis.md

```diff
@@ -0,0 +1,179 @@
+---
+title: Interact with Moralis APIs
+description: Learn how Moralis' API suite empowers developers to retrieve and leverage various data sets from Moonbeam, Moonriver, and Moonbase Alpha.
+---
+
+# Access Moonbeam Data via Moralis APIs
+
+## Introduction {: #introduction }
+
+As a one-stop solution for blockchain development, [Moralis](https://moralis.io/){target=\_blank} offers a comprehensive platform that empowers developers to create, launch, and scale decentralized applications (dApps) with ease.
```

Comment:
Feels like we could use maybe 1 more sentence here introducing Moralis before diving into the features.